src/models.c

```diff
@@ -2650,25 +2650,47 @@ BoundingBox GetMeshBoundingBox(Mesh mesh)
 // Compute mesh tangents
 // NOTE: To calculate mesh tangents and binormals we need mesh vertex positions and texture coordinates
 // Implementation base don: https://answers.unity.com/questions/7789/calculating-tangents-vector4.html
-void GenMeshTangents(Mesh *mesh)
+void GenMeshTangents(Mesh* mesh)
```

Comment:
Following [raylib code style](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md) I place the asterix next to the variable instead of the type.